### PR TITLE
WL-4274 Created calendar component for Lessons page.

### PR DESCRIPTION
**Pull request by ouit0408**
New option 'Embed Calendar' is added in the list on Lessons page.
SimplePageItem of type 'Calendar' created for calendar widget.
Calendar shows events from all the merged calendars in the Lessons tool.
If event has hyperlink as description they are shown as hyperlink, same
applies for embed code.
This PR is linked to https://github.com/ox-it/wl-calendar/pull/19

### CODE REVIEW

**Comment by buckett** [comment]
Doesn't merge.

File: tool/src/webapp/js/lessons-calendar.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                    });
+                },
+                color: '#D4DFEE',
+                textColor: '#0064cd'
+            }
+        ],
+        eventClick: function(event, jsEvent, view){
+            var startDate = new Date(event.start);
+            var endDate = new Date(event.end);
+            $("#startTime").text(moment(startDate).format('DD-MM-YYYY hh:mm A'));
```
Comment (buckett) [inline tool/src/webapp/js/lessons-calendar.js]: How are we dealing with i18n here?

**Comment by buckett** [comment]
Doesn't merge again.

**Comment by ouit0408** [comment]
Added user's locale for the start and end date, have pushed the changes.

**Pull request merged**